doc/dftb+/manual/dftbp.tex

```diff
@@ -6392,6 +6396,16 @@ \subsection{qsvst.dat}
 where N is the number of atoms, and the net total charge should always
 be zero (up to numerical presicion) for a neutral system.
 
+\subsection{atomenergies.dat}
+\index{atomenergies.dat}
+
+Atom-wise contribution to total energy (without nuclear kinetic energy)
```

Comment:
Is there a reason not to include Ekin in the output, or even give the energy with it included in a separate output?

Reply:
It can be calculated with the velocities printed to tdcoords.xyz, and the user that requested this feature does not seem to need the Ekin per atom, so I decided to leave it out for simplicity